.github/dependabot.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+# Dependabot config that checks version updates for go.mod packages and docker
+# images, and also checks only for security updates for GitHub actions.
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    # Disable version updates and only check security updates for GitHub
+    # actions, since we can't bump the versions until they're on our allow-list
+    open-pull-requests-limit: 0
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
FWIW I don't think this will do much until the Dockerfile uses explicit tags in the FROM lines for dev and default. The indirection with $ALPINE_VERSION fools dependabot.

Reply:
Thanks. Put up #248  to address that.